test/system/030-run.bats

```diff
@@ -1158,4 +1158,15 @@ EOF
     run_podman rm -f -t0 $ctr
 }
 
+@test "podman run --authfile=/tmp/bogus " {
+    run_podman 125 run --authfile=/tmp/bogus $IMAGE false
```

Comment:
Could we use `$PODMAN_TMPDIR/bogus-authfile` instead? (preferably declaring it as a var). It is not inconceivable for a `/tmp/bogus` file to exist on someone's system.